src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: flex;
+    color: #060b35;
+    text-decoration: none;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    text-transform: uppercase;
+    line-height: 60px;
+    height: 60px;
+    margin-left: 20px;
```

Comment:
The `display: flex;` style applied to links seems redundant since you don't have any items inside the links that would require a flex container. Also, the `height: 60px;` style might not be necessary if you adjust your line-height or padding properly.